### PR TITLE
Add blog pages

### LINKED ISSUE #18: Create Blog > Articles page
Issue to track creation of Blog Articles page.

NB: currently on the Miro board we only have a "Workshops" tab under this heading. Suggest we have both Blog>Articles and Blog>Workshops pages.

Can port across the old blog entried from here: https://gstt-csc.github.io/blog.html

Suggest having a more traditional blog type page, rather than a markdown table of links, i.e.: more like the posts layout here: https://mmistakes.github.io/so-simple-theme/

First new blog post can be introducing the new site!

### PR DESCRIPTION
**Pull request by AnilMistry**
I have created blog cards here for articles and workshops, which i think looks better than displaying in a list or having a continuous scroll blog. These model cards only source their data from data/blogs.yml, which then provides a link to markdown file. 

- I kept all markdowns in the root directory as it seems like a massive headache to use subdirectories in jekyll. 
- The "metadata" in the markdown is almost a direct copy of what needs to be added to the data/blog.yml, which I'm going to argure is a positive. 
- I could hyperlink the authors picture in the blog to the CSC team members profile in future, but would need some conditioning incase the author isnt in the CSC core team e.g. trainees. 

resolves #18 and #19

### CODE REVIEW

File: _includes/blog-articles.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!-- Articles -->
+<div class="container">
+  <div class="row pt-3">
+  {% for blog in site.data.blogs[site.locale].articles %}
+	<div class="col-sm-4 pb-5" id="blogCard" >
+	<a href="{{blog.blog-page}}">
+	  <div class="card">
+		<div class="card__header">
+		  <img src="{{blog.image}}" alt="card__image" class="card__image" width="350">
```
Comment (tomaroberts) [inline _includes/blog-articles.html:9]: Any reason for the double underscore "__" throughout? Single underscore would be better coding convention.

**Review by tomaroberts — CHANGES_REQUESTED** [review]
This looks great Anil. There are a few small stylistic things I would suggest changing, but I won't go into huge detail for now as we just want to get the site together at this stage.

Points for now:

1) We really need to put the blog posts in a subdirectory called `/blog` or `/posts`. The root directory is filling up quickly with lots of files. I had a quick look into this, but can't figure it out right away.

Jekyll has the `_posts` folder functionality which is designed for blogging purposes: https://jekyllrb.com/docs/posts/. Here's an example implementation on another open-source Jekyll theme: https://github.com/mmistakes/so-simple-theme/tree/master/example

Perhaps we could implement something along these lines?

2) It would be good if the blog cards were the same height

3) Similarly, when you click through it would be good if the top blogpost image filled the column width you've created, e.g. this image isn't wide enough – looks like your column is 760px wide?:  

image

**Comment by tomaroberts** [comment]
I'm going to merge these changes in btw, but also keep the branch open for ongoing refinement.

File: _includes/blog-articles.html (hunk 1 above)
Comment (AnilMistry) [reply _includes/blog-articles.html:9]: No real reason, probably an oversight when using publicly available code as     inspiration.

**Comment by AnilMistry** [comment]
Cheers @tomaroberts I'll tackle those points this week. I think subfolder should be addressed as soon as possible before the website grows. 
I agree with the card formatting issues, I'll tackle them on this branch.

**Pull request merged**